imu.py

```diff
@@ -0,0 +1,28 @@
+"""
+  Simple IMU data analysis
```

Comment:
Is this going to be a tool or library?

Reply:
Well, it is both :(. It is like laser.py, camera.py or gps.py, i.e. sensors which can be attached to various machines (if we reserve word "robot" for already integrated system, like John Deere with laser + camera + gps). I will probably split this into separate branch/PR.